configs/config-suite-b.h

```diff
@@ -114,8 +114,8 @@
 //#define MBEDTLS_USE_PSA_CRYPTO
 //#define MBEDTLS_PSA_CRYPTO_C
 
-/* With USE_PSA_CRYPTO, some PK operations also need PK_WRITE */
-#if defined(MBEDTLS_USE_PSA_CRYPTO)
+/* With MBEDTLS_PSA_CRYPTO_C, some PK operations also need PK_WRITE */
```

Comment:
Nope, that one was indeed `USE_PSA_CRYPTO`: this config doesn't have RSA, only ECDSA, and with ECDSA the dependency on `PK_WRITE` is in the version of `ecdsa_verify_wrap` that's guarded by `MBEDTLS_USE_PSA_CRYPTO`.